docs/pass-through.md

```diff
@@ -0,0 +1,237 @@
+# Running Antrea In Pass-Through Mode
+
+Antrea supports chaining with other CNI implementations such as EKS, AKS. In this mode, 
+Antrea enforces k8s network policies, and delegates Pod IP management
+and network connectivity to the primary CNI.
+
+The pass-through mode is designed to be compatible with all major CNI implementations, 
+with emphasis to support cloud provider managed clusters such asAKS, EKS, or potentially
+some flavors of GKE clusters.  
+
+## Deploy and Test Antrea In Cloud 
+Assuming you already have a cloud managed k8s cluster, and have KUBECONFIG environment
+variable set to configuration file of that k8s cluster. You may also refer to [here](cloud.md)
+to create an AKS or EKS cluster on the flight.
+
+Create an Antrea manifest for pass-through mode, for instance
+```bash
+./hack/generate-manifest.sh --encap-mode passThrough > ~/my-yaml/antrea-passthrough.yaml
+```
+
+Adjust defaultMTU, and serviceCIDR values of antrea-agent.conf in the manifest to your cluster, and 
+apply the manifest to the cluster, for instance
+```bash
+kubectl apply -f ~/my-yaml/antrea-passthrough.yaml 
+```
+
+Now Antrea should be plugged into the k8s cluster and enforces any network policies.
+You can now validate Antrea network policy engine by run k8s conformance and network policy tests.
+```bash
+./ci/run-k8s-e2e-tests.sh --e2e-all
+```
+You may need to install a lower version of sonobuoy 
+[here](https://github.com/vmware-tanzu/sonobuoy/releases/), and set environment variable SONOBUOY to
+path to the executable, if you k8s cluster is lower version;
+
+
+### Caveats
+When install Antrea after a k8s cluster already has some Pods running, these pre-installed Pods
```

Comment:
suggestion for this whole paragraph:

When installing Antrea on a K8s cluster already running some Pods, Antrea will not be able to enforce network policies on these existing Pods. If Pods are restarted, then Antrea can enforce network policies on them. For instance, to restart all Pods in the coreDNS deployment, you can do the following: